elixir-test.el

```diff
@@ -104,7 +108,9 @@ file, or the whole test suite, respectively."
 		     location))
 	 (test-cmd (vector base-cmd args location)))
     (elixir-test-set-last-test default-directory test-cmd)
-    (compile (elixir-test-format-command test-cmd))))
+    (compilation-start (elixir-test-format-command test-cmd)
+		       'elixir-test-output-mode
+		       (lambda (mode) (concat "*" mode " " (elixir-test--project-name default-directory) "*")))))
```

Comment:
I'm going to look like a completely insane person talking to myself in my own PR, but this line is hella long.

Reply:
Could potentially break it out into a real, named function instead of a lambda.